packages/roosterjs-editor-dom/lib/utils/applyTableFormat.ts

```diff
@@ -13,16 +19,17 @@ const CELL_SHADE = 'cellShade';
 export default function applyTableFormat(
     table: HTMLTableElement,
     cells: VCell[][],
-    format: Partial<TableFormat>
+    format: Partial<TableFormat>,
```

Comment:
For this function, it should accept Required`<TableFormat>`

For the functions face to other developer, we can loose the type and accept `Partial<>`, but for internal API that only called by our code, we need to make it strong-typed and only accept well-formed data.

Reply:
insertTable and formatTable will call VTable.applyFormat(), right? Then VTable.applyFormat() can take the responsibility to full fill this object